src/Symfony/Component/HttpKernel/Event/FilterControllerArgumentsEvent.php

```diff
@@ -17,7 +17,7 @@
 /**
  * @deprecated since Symfony 4.3, use ControllerArgumentsEvent instead
  */
-class FilterControllerArgumentsEvent extends ControllerEvent
+class FilterControllerArgumentsEvent extends FilterControllerEvent
```

Comment:
why this change?

Reply:
So we can make ControllerEvent final. We just need to copy the setController and getController to the ControllerArgumentsEvent in 5.0

Reply:
thanks